internal/host/transition_test.go

```diff
@@ -1463,62 +1463,72 @@ var _ = Describe("Unbind", func() {
 	tests := []struct {
 		name      string
 		srcState  string
+		dstState  string
 		success   bool
 		sendEvent bool
 	}{
 		{
 			name:      "known to unbinding",
 			srcState:  models.HostStatusKnown,
+			dstState:  models.HostStatusUnbinding,
 			success:   true,
 			sendEvent: true,
 		},
 		{
 			name:      "disabled to unbinding",
 			srcState:  models.HostStatusDisabled,
+			dstState:  models.HostStatusUnbinding,
 			success:   true,
 			sendEvent: true,
 		},
 		{
 			name:      "disconnected to unbinding",
 			srcState:  models.HostStatusDisconnected,
+			dstState:  models.HostStatusUnbinding,
 			success:   true,
 			sendEvent: true,
 		},
 		{
 			name:      "discovering to unbinding",
 			srcState:  models.HostStatusDiscovering,
+			dstState:  models.HostStatusUnbinding,
```

Comment:
@yevgeny-shnaidman WDYT?

Reply:
currently host always becomes unbinding, and when the nodes will reconnect it will move to discovering-unbound. Maybe we need to fix it, but it needs to be done in a different PR, since we need to move it to a different transaction